iTunes Media App/Controllers/SongsViewController.swift

```diff
@@ -73,12 +53,16 @@ extension SongsViewController: UITableViewDelegate, UITableViewDataSource {
     }
     
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        let cell = tableView.dequeueReusableCell(withIdentifier: SongTableViewCell.reuseID, for: indexPath)
+        let cell = tableView.dequeueReusableCell(withIdentifier: SongTableViewCell.reuseID, for: indexPath) as! SongTableViewCell
         let song = songs[indexPath.row]
-        
-        cell.textLabel?.text = song.name
-        
-        
+
+        // Added Accessory Button
+        let imageView: UIImageView = UIImageView(frame:CGRect(x: 0, y: 0, width: 20, height: 20))
```

Comment:
declared the CGRect in its own line, it's easier to read.
```suggestion
        let imageFrame = CGRect(x: 0, y: 0, width: 20, height: 20)
        let imageView: UIImageView = UIImageView(frame: imageFrame)
```

Reply:
Definitely looks cleaner. Thanks!